server/src/callback.js

```diff
@@ -0,0 +1,18 @@
+function eventCallback(req, res) {
+  // check auth
+  if (req.header('Bearer')) {
+    console.log(req.header('Bearer'));
+    console.log(req.header('Authorization'));
+  }
+
+  // get the base handler to add the event
+  // send notifications (probably belongs in the other code)
+  res.send('OK');
+}
+
+export default function getCallback(type) {
```

Comment:
i dont really understand the need for this. Couldn't you just `app.use()` and pass the `eventCallback` function directly?